source/common/formatter/substitution_format_utility.cc

```diff
@@ -85,6 +85,15 @@ void SubstitutionFormatUtils::truncate(std::string& str, absl::optional<size_t>
   }
 }
 
+absl::string_view SubstitutionFormatUtils::truncateStringView(const absl::string_view& str,
```

Comment:
Thanks for your contribution, this looks like a good patch.

You need not to use const reference of string view (const absl::string_view&) as the input. Please use the `absl::string_view` directly.

And please check the CI. Thanks.

/wait